dts/bindings/ethernet/atmel,sam-gmac.yaml

```diff
@@ -41,6 +41,16 @@ properties:
       description: |
         Operation mode of the PHY interface.
 
+    peripheral-id:
+      type: int
+      default: 0
+      description: |
+        A peripheral identifier is required for the control of the peripheral
+        interrupt with the Nested Vectored Interrupt Controller (NVIC) and
+        control of the peripheral clock with the Power Management Controller
+        (PMC).  This parameter is mandatory for any SAM family.  SAM0 family
+        uses another strategy to define both interrupt and clock control.
```

Comment:
I'm confused about this. It says it's mandatory but it has a default value, which doesn't make sense.